events/ecommerce/wishlist_viewed.md

```diff
@@ -0,0 +1,26 @@
+# View Wishlist
+
+Send when a user views their wishlist.
+
+## Javascript Code
+
+```js
+// When:
+// User user views their wishlist
+
+// Code:
+window.dataLayer = window.dataLayer || [];
+dataLayer.push({ ecommerce: null });  // Clear the previous ecommerce object.
+dataLayer.push({
+  event: "wishlist_viewed",
```

Comment:
suggest `view_wishlist` instead. consider other `view_` events in GA4 spec:
- `view_cart`
- `view_item`
- `view_item_list`
- `view_promotion`

regardless of how DCL implements, this should reflect what we want as canonized in the spec. @jcottriel-sdi thoughts?

Reply:
I'm happy to change this around to view_whislist.